src/osmf/js/osmf_flash.js

```diff
@@ -547,7 +570,10 @@ require("../../../html5-common/js/utils/constants.js");
     };
 
     var raiseVolumeEvent = function(event) {
-      newController.notify(newController.EVENTS.VOLUME_CHANGE, { "volume" : event.eventObject.volume });
+      var volume = event.eventObject.volume;
+      var muted = volume === 0;
+      newController.notify(newController.EVENTS.VOLUME_CHANGE, { "volume" : volume});
```

Comment:
Is there a reason why "volume" is in quotes but muted is not?

Reply:
Nope, I just failed at consistency :( It doesn't break anything though, can this be addressed in another PR?